src/components/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,29 @@
+import "./DropDown.scss";
+import topicArrow from "../../assets/upside-down-arrow-grey.svg";
+
+const DropDown = ({ topicTitle, lessonTitle, lessonType }) => {
```

Comment:
We need to pass down a list of lesson titles, and map through them to display few of them and not just, don't worry about lesson types for now, just display the text without it